src/status_im/notifications_center/core.cljs

```diff
@@ -56,13 +56,14 @@
 (fx/defn accept-activity-center-notifications
   {:events [:accept-activity-center-notifications]}
   [{:keys [db]} ids]
-  {:db (update-in db [:activity.center/notifications :notifications]
-                  (fn [items] (remove #(get ids (:id %)) items)))
-   ::json-rpc/call [{:method     (json-rpc/call-ext-method "acceptActivityCenterNotifications")
-                     :params     [ids]
-                     :js-response true
-                     :on-success #(re-frame/dispatch [:sanitize-messages-and-process-response %])
-                     :on-error   #()}]})
+  (when-not (empty? ids)
```

Comment:
that's just a recommendation but anyway https://github.com/bbatsov/clojure-style-guide#nil-punning